packages/volto-accordion-block/src/index.js

```diff
@@ -1,11 +1,26 @@
-import accordionSVG from '@plone/volto/icons/list-arrows.svg';
+import accordionSVG from '@plone/volto/icons/circle-bottom.svg';
+import imagesSVG from '@plone/volto/icons/images.svg';
+import TestimonialIcon from '@plone/volto/icons/quote.svg';
 import Edit from './components/Edit';
 import View from './components/View';
 import layoutSchema from './components/LayoutSchema';
 import { AccordionStylingSchema } from './components/schema';
 import PanelsWidget from './components/Widgets/PanelsWidget';
-// import filterSVG from '@plone/volto/icons/filter.svg';
-// import clearSVG from '@plone/volto/icons/clear.svg';
+
+//FHNW Blocks
+import { ImageSliderDataAdapter } from '@fhnw/components/ImageSlider/adapter.js';
```

Comment:
@Tishasoumya-02 we cannot refer to FHNW code in the addon.
If afterwards, there is a customization happening in the project, it has to be there where it happens, not in the addons.